samples/HttpClientFactorySample/Program.cs

```diff
@@ -23,7 +26,16 @@ public class Program
                     c.DefaultRequestHeaders.Add("Accept", "application/vnd.github.v3+json"); // Github API versioning
                     c.DefaultRequestHeaders.Add("User-Agent", "HttpClientFactory-Sample"); // Github requires a user-agent
                 })
-                .AddLogging()
+
+                // Retry requests to github using our retry handler
+                .AddHttpMessageHandlerPolicy("github", DefaultStages.Reliability, b => b.AdditionalHandlers.Add(new RetryHandler()))
```

Comment:
This goes away right?

Reply:
> DefaultStages.Reliability 

That part does. What do you mean specifically?

Reply:
Thats what I meant, I pulled the branch and realized it doesn't build so that just needed to be updated. I'm also not a fan of the top level method on service collection to add message handlers. I'm wondering if a builder pattern would be better.

Reply:
Yeah, I think we will need the builder.